http/file_server.ts

```diff
@@ -517,20 +517,22 @@ function dirViewerTemplate(dirname: string, entries: EntryInfo[]): string {
       <body>
         <main>
           <h1>
-            Index of ${headerPaths
-              .map((path, index) => {
-                if (path === "") return "";
-                const depth = headerPaths.length - index - 1;
-                let link;
-                if (depth == 0) {
-                  link = ".";
-                } else {
-                  link = "../".repeat(depth);
-                }
-                // deno-fmt-ignore
-                return html`<a href="${link}">${escape(path)}</a>`;
-              })
-              .join("/")}/
+            Index of ${
```

Comment:
It seems these formatting changes were made because your Deno version is out-of-date. Please update Deno and run `deno fmt` to revert these changes.